src/core/internal/traits.d

```diff
@@ -210,3 +210,23 @@ template Filter(alias pred, TList...)
                 Filter!(pred, TList[$/2 ..  $ ]));
     }
 }
+
+// std.meta.staticMap
```

Comment:
Looking at the documentation for std.meta.staticMap, all it says is:
```
Evaluates to $(D AliasSeq!(F!(T[0]), F!(T[1]), ..., F!(T[$ - 1]))).
```
but that is not what this template body does.